backend/app/adapter/routing/handle/search.go

```diff
@@ -47,6 +48,11 @@ type SearchResponse struct {
 	Users      []User      `json:"users,omitempty"`
 }
 
+// SearchError represents an error with the Search API request.
+type SearchError struct {
+	Message string `json:"message"`
+}
```

Comment:
I wonder what is this for? Are we going handle it in the frontend?

Reply:
This will provide information about the search error that occurred. So far, we have "user not provided" and "unknown resource" errors. I think it would make sense to emit these errors in the frontend, even though they may only show up in rare circumstances (e.g. user somehow gets logged out right before they search for something). What do you think?